yt_dlp/extractor/rai.py

```diff
@@ -704,19 +706,29 @@ def _real_extract(self, url):
             except ExtractorError:
                 pass
 
-        relinker_url = self._proto_relative_url(self._search_regex(
-            r'''(?x)
-                (?:
-                    var\s+videoURL|
-                    mediaInfo\.mediaUri
-                )\s*=\s*
-                ([\'"])
-                (?P<url>
-                    (?:https?:)?
-                    //mediapolis(?:vod)?\.rai\.it/relinker/relinkerServlet\.htm\?
-                    (?:(?!\1).)*\bcont=(?:(?!\1).)+)\1
-            ''',
-            webpage, 'relinker URL', group='url'))
+        # new rainews player (#3911)
+        player_data = self._parse_json(
```

Comment:
Use _search_json

Reply:
You can add `transform_source=clean_html`

Reply:
I haven't touched code for a long time. don't remember a lot of things ;-)

thanks for the patience.